src/app/pages/wallets/wallet-details/pages-wallet-details.component.scss

```diff
@@ -53,4 +55,50 @@
     opacity: 0.75;
     font-size: 0.75em;
   }
+}
+
+@media only screen and (max-width: 500px) {
```

Comment:
All looks good but you should use media queries in "opposite direction". First you should define styles for mobile and then when needed use `min-width` to change them for bigger and bigger resolutions :)